leftwm/src/config.rs

```diff
@@ -96,6 +86,46 @@ pub struct WindowHook {
     pub hiding_strategy: Option<WindowHidingStrategy>,
 }
 
+#[derive(Debug, Clone)]
+pub struct SerializableRegex {
+    regex: Regex,
+}
+
+impl SerializableRegex {
+    /// # Panics
+    ///
+    /// Function can only panic if the str used is not a valid regex
```

Comment:
Probably should call out for reference which syntax to use; think it's still https://github.com/google/re2/wiki/Syntax, no?

Reply:
I think that maybe instead of using a github page with the definition, it might be easier to link the docs from the create that is used ? https://docs.rs/regex/latest/regex/#syntax

Reply:
Works even better! Great idea.